agenda/presentation/src/main/java/com/pronoidsoftware/agenda/presentation/detail/reminder/ReminderDetailScreen.kt

```diff
@@ -70,6 +71,7 @@ fun ReminderDetailScreenRoot(
 
     ReminderDetailScreen(
         state = viewModel.state,
+        clock = viewModel.clock,
```

Comment:
Just a spontaneous thought I had, maybe it could make sense to make this a composition local which makes the clock accessible from any composable with `LocalClock.current`. Initially you could provide it from your activity wide ViewModel 🤔 
Don't see any downsides right now, even though composition locals should be used with care

Reply:
ah, interesting thought, updated the PR with this suggestion